docs/multiform.rst

```diff
@@ -60,6 +60,13 @@ use an OrderedDict to define the :attr:`~MultiForm.form_classes`. ::
             ('profile', ProfileForm),
         ))
 
+An implementation of the attribute :attr:`~MultiForm.base_fields` is provided,
+but the keys are a combination of the subform keys and the field names rather
+than just the field names. The reason an implemention is provided at all, given
+it is merely an approximation, is interoperability with ``WizardView``, which
+introspects the values of :attr:`~MultiForm.base_fields`.  See
+:ref:`wizardview_guide`.
+
```

Comment:
Maybe, but isn't it django-formtools' responsibility to worry about that? I'm pretty runtime modification also scuppers WizardView's introspection for a normal Django Form. Doesn't such a note, or perhaps a fix to check fields as well as base_fields, properly belong in django-formtools?